src/Views/Reference/NavalFlagTable.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { NavalFlags, CharacterImage } from 'puzzle-lib';
+import { Table } from 'react-bootstrap';
+import './NavalFlagTable.css';
+
+class NavalFlagTable extends React.Component {
+  public render() {
+    return (
+      <div className="NavalFlagsTable">
+        <div className="NavalFlagsTable-content">
+          <Table striped={true} responsive={true}>
+            <thead>
+              <tr>
+                <th>Letter</th>
+                <th>Flag</th>
+              </tr>
+            </thead>
+            <tbody>
+              {this.renderRows(NavalFlags.instance.entries)}
+            </tbody>
+          </Table>
+        </div>
+      </div>
+    );
+  }
+
+  private renderRows(data: CharacterImage[]) {
+    return data.map((value: CharacterImage) => {
+      return (
+        <tr key={value.character}>
+          <td className="flagCharacter">{value.character}</td>
+          <td dangerouslySetInnerHTML={{__html: value.image.render(100)}} />
```

Comment:
Consider this instead of "dangerouslySetInnerHtml" https://www.npmjs.com/package/react-inlinesvg

Reply:
I'd recommend using a [data URI](https://css-tricks.com/probably-dont-base64-svg/) instead.

Here's my proposed change: https://github.com/kfarnung/puzztool/commit/2c093365004fdb5e5a933e27113bfdc549c3964a

It requires https://github.com/kfarnung/puzzle-lib/pull/66 to land first though.

Reply:
Done.  We can potentially drop the width parameter from the render function entirely.

Reply:
Opened a PR to drop the width parameter: https://github.com/kfarnung/puzzle-lib/pull/67

Suggested updates: https://github.com/kfarnung/puzztool/commit/b2ac463a9e1af704c3c1172b2d1d0810290ebd45